src/pdu_loop/storage.rs

```diff
@@ -321,7 +321,11 @@ mod tests {
             .unwrap();
 
         // Drop frame future to reset its state to `FrameState::None`
-        drop(frame.mark_sendable(&pdu_loop, Duration::MAX, usize::MAX));
+        let max_timeout = crate::timer_factory::LabeledTimeout {
+            duration: Duration::MAX,
+            kind: crate::timer_factory::TimeoutKind::Pdu,
+        };
+        drop(frame.mark_sendable(&pdu_loop, max_timeout, usize::MAX));
```

Comment:
This could use `MAX_TIMEOUT` I think? 

```suggestion
        drop(frame.mark_sendable(&pdu_loop, MAX_TIMEOUT, usize::MAX));
```